schedule/yam/test_cases/migration_mu.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
'mu' is internal Yam abbreviation, please not use it. use something more descriptive and in line with current name convention.
Also our /test_cases/ folder should not contain files which are not yet in yaml based in default, should be our pristine folder, add this somewhere else until further refactor.

Reply:
I am not sure what is the yaml schedule convention we have agreed to follow. I changed the name and location to schedule/migration/migration_autoyast.yaml . I am trying to give a relatively generic name because it seems that the schedule can be used in other scenarios. WDYT?

Reply:
I would go with something more specific, in the future we will make migrations more modular, so the schedule will differ like in YaST and we will have to use yaml based on default, but it is ok, you don't need to change this, the name you use it is what does :)